server.py

```diff
@@ -35,8 +36,8 @@ class InvestigateRequest(BaseModel):
     title: str
     description: str
     subject: dict
-    context: List[InvestigateContext]
```

Comment:
if we dont use InvestigateContext anymore we should get rid of the class from above

Reply:
nice catch, fixing